vite.config.ts

```diff
@@ -91,6 +91,15 @@ export default defineConfig({
   },
   plugins: [
     dts({
+      afterBuild: async () => {
+        await fs.writeFile(
+          'dist/index.d.ts',
+          (await fs.readFile('dist/index.d.ts'))
+            .toString()
+            .replace(/\nexport .+/, '')
+             + "export = _default"
+        )
+      },
```

Comment:
This is admittedly kind of hacky 😞 but I couldn't figure out a way to get this through the existing tooling.

* Switching the `tsconfig.json` `module` to `"nodenext"` doesn't change anything
* Neither does `"commonjs"` with a removal of `verbatimModuleSyntax`
* `vite-plugin-dts` doesn't have options for dual-format-emit that I could find